keras/backend/theano_backend.py

```diff
@@ -1488,6 +1488,9 @@ def rnn(step_function, inputs, initial_states,
             raise ValueError('When specifying `unroll=True`, '
                              'an `input_length` '
                              'must be provided to `rnn`.')
+        if input_length == 1:
```

Comment:
Is this error actually necessary? Couldn't we allow input_length=1 and unroll=True? (loop over 1 element).

Reply:
The test fails for theano backend. I could definitely fix it, just didn't want to invest time in theano backend. Do you really want it?

Reply:
Got it. Since the error is specific to Theano, it seems fine.